dns_sd_windows.c

```diff
@@ -20,6 +20,10 @@
 #include "iio-private.h"
 #include "deps/mdns/mdns.h"
 
+#define _STRINGIFY(x) #x
+#define STRINGIFY(x) _STRINGIFY(x)
+#define MDNS_PORT_STR _STRINGIFY(MDNS_PORT)
```

Comment:
Took me a few days to debug... But the fix is literally removing *one* character:
```
#define MDNS_PORT_STR STRINGIFY(MDNS_PORT)
```

One thing I wonder, is the usefulness to list scanned contexts on both IPv6 and IPv4; because afterwards, there is no way to specify which to use (they both would list the URI as `ip:analog.local`).

Reply:
fixed on the branch, just testing to see if this resolves things.